softdev.bib

```diff
@@ -2759,15 +2759,15 @@ @misc{watson21assessing
 }
 
 @misc{servo,
-    title = {The Servo Parallel Browser Engine Project},
+    title = "The {Servo} {Parallel} {Browser} {Engine} {Project}",
     howpublished = {\href{https://servo.org}{https://servo.org}},
     note = {Accessed: 2022-12-17},
     year = {2012},
     author = {Mozilla},
 }
 
 @misc{rustgc,
-    title = {rust-gc: simple tracing (mark and sweep) garbage collector for Rust},
+    title = "rust-gc: a simple tracing (mark and sweep) garbage collector for {Rust}",
```

Comment:
If `rust-gc` needs to be lower-case... you'll have to bracket it too!

Reply:
When I do that I get a prebib syntax error which I can't decipher:
```
I was expecting a `,' or a `}'---line 2497 of file out.bib
 :     title = {rust-gc}
 :                      : a simple tracing (mark and sweep) garbage collector for {Rust},
```
Note I am actually including the double quotes "", which seem to be missing from the prebib warning. 

Coincidentally, pdflatex seems to format this as I intended.

Reply:
Sounds like a prebib bug to me? Maybe there's a quick fix to prebib?

Reply:
Ah, it turns out of you have curlys at the beginning and end of a title, you need to wrap the entire title in "" and curlys to keep the bibtex parser happy... Don't ask f7a6f5705ba1f01cc5a2c9365d5e4b0ed54111a8